src/containers/Projects/ProjectDetails/constants.ts

```diff
@@ -33,12 +33,6 @@ export const projectDetailsTopic: ProjectTopicMap = {
     position: 1,
```

Comment:
Hey, `position` is not being used now I think. Can we also get rid of these in this PR?

Reply:
It is being used by the `ProjectDetailsSideMenu` component which uses the ordered version of the project topics.

Reply:
🤔 that could have been achieved w/o position too, if we disable eslint rule on that file?

Reply:
@thesanjeevsharma technically its unreliable to sort objects, although in practice it'll always be ordered the way we want it (after we disable es-lint). But just out of principle, i never like depending on the order of an object alone to dictate it's order